app/src/androidTest/java/org/mozilla/fenix/ui/CollectionTest.kt

```diff
@@ -258,14 +254,14 @@ class CollectionTest {
         }
     }
 
-    @Ignore("Failing, see: https://bugzilla.mozilla.org/show_bug.cgi?id=1812580")
     @Test
     fun verifyAddTabButtonOfCollectionMenu() {
         val firstWebPage = getGenericAsset(mockWebServer, 1)
         val secondWebPage = getGenericAsset(mockWebServer, 2)
 
         navigationToolbar {
         }.enterURLAndEnterToBrowser(firstWebPage.url) {
+            mDevice.waitForIdle()
```

Comment:
This would help fix https://console.firebase.google.com/u/1/project/moz-fenix/testlab/histories/bh.66b7091e15d53d45/matrices/4786210184261246377/details?stepId=bs.310f2b1a64a45f6&testCaseId=1
>  androidx.test.espresso.PerformException: Error performing 'single click - At Coordinates: 893, 720 and precision: 16, 16' on view 'view.getId() is <16908313/android:id/button1>'. 
> ...
>  Caused by: java.lang.IllegalStateException: An anchor may not be changed after CoordinatorLayout measurement begins before layout is complete.

Reply:
We usually prefer UiObject.waitforExists(waitingTime) instead of waitforIdle which proved not so useful.